modules/microshift-openshift-apis.adoc

```diff
@@ -16,3 +16,19 @@ In addition to the standard Kubernetes APIs, {product-title} includes a subset o
 | xref:../microshift_rest_api/security_apis/securitycontextconstraints-security-openshift-io-v1.adoc#securitycontextconstraints-security-openshift-io-v1[SecurityContextConstraints]
```

Comment:
I know this is out of scope of this PR, but I know we can't have xrefs in modules. So this might be something that needs to be fixed later

Reply:
Opp edit: Just found this jira that covers this issue https://issues.redhat.com/browse/OSDOCS-4710
so ignore this 😄